app/views/govuk_publishing_components/components/_attachment.html.erb

```diff
@@ -4,10 +4,14 @@
   hide_help_text ||= false
   attributes = []
 
-  if attachment.content_type_abbr
-    content = tag.abbr(attachment.content_type_abbr,
-                       class: "gem-c-attachment__abbr",
-                       title: attachment.readable_content_type)
+  if attachment.readable_content_type
```

Comment:
Appreciate it's not part of this part, but 'readable' confused me for a while until I realised it means 'human_readable' :-|.

Reply:
I'm not particularly keen on the readable method names here. I don't really think human_readable helps either though as it gets a bit whacky when it's human_readable_number_of_pages 🤔 It might be that it's best to just do the view helper transformations in the view rather than presenter.

Reply:
Ah, I see what you mean. Looking at it again, it seems the 'readable' prefix is somewhat unnecessary - there's no (unreadable ;-) number_of_pages, for example. Perhaps it could more usefully be 'content_type_name', as with 'content_type_abbr'. Up to you, though :-).

Reply:
I've dropped the readable_* methods now